### DISCUSSION TITLE
[Simulation question]: Chemical stage time end in SBS

### DISCUSSION THREAD

**Discussion by ShahinGHD** [topas-nbio/TOPAS-nBio#52] [Simulation-question] [answered]
### Simulation question

Hi everyone.

My question is about this line in my step-by-step simulations: 
d:Ch/TOPASChemistry/ChemicalStageTimeEnd = 1 ns

I have a circular plasmid with 100 base pairs in a box and I have a point source Lu-177 at the center undergoing disintegrations.

My question is: what is the best value for this parameter?
I set it now to 1 ns, does it mean that the chemical species will be produced from 0 to 1 nanosecond and after that, it starts to diffuse to my DNA? or it means it will track the chemical species diffusion for 1 ns?
because from what I have tested, by decreasing this value I get less indirect DNA damage.
another thing that I saw is that when i increase it, i get more lines in the out put file as well as more -nan values.

it is a bit confusing to me as in the manual it says  "The end time must be larger than 1 ps (i.e. the time where the chemical stage
begins)." 

Thank you a lot in advance.

### TOPAS-nBio category

Physics

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
That parameter dictates the end of the chemistry processes which includes Brownian motion and reactions between chemical species. The products from radiolysis of water (pre-chemical stage) are put in the simulation at 1 ps, thus the time end for the chemistry processes that follow should be greater than 1 ps.

Decreasing the time end will cut the transport of chemical species before they reach the plasmid.

An example. For your specific application, the time end depends on the scavenging capacity _s_ of the medium (_s = k [B]_) and the chemical species of interest (e.g., OH) that this medium scavenges. The inverse of _s_ gives you the time at which the concentration of the scavenged species has dropped to _1/e_. Thus, for a cellular-like medium with scavenging capacity of c.a. 10^9 /s, the end time is about c.a. ns.  For a plasmid suspended in, e.g., DMSO (scavenger of OH, with k~7e9 /M/s), the concentration should be about 0.1 M to mimic roughly the cellular scavenging capacity and thus require the same time end.

Therefore, the time end depends on your experimental conditions: type of scavenger and its concentration.
Cheers.

**Comment by ShahinGHD** [comment]
Thank you Jose I always appreciate your replies. 

one quick question: The parameter

` u:Sc/DBSCAN/SampleHitsWithProbability   = 0.16    # Hits are scorer with 16% probability`

in the DBSCAN method, does this mean it is considered that 16% of the scoring volume is occupied by the DNA? (since the DNA geometry is not important in the DBSCAN method)

I am trying to have a same volume in both SBS and DBSCAN method to see if I can get similar "direct" DNA damages.

Thanks.
